docs/releases/v2023.1.rst

```diff
@@ -0,0 +1,120 @@
+Gluon 2023.1
+============
+
+Important notes
+---------------
+
+Upgrades to v2023.1 and later releases are only supported from releases v2021.1 and later.
+This is due to migrations that have been removed to simplify maintenance.
+
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- Extreme Networks
+
+  - WS-AP3805i
+
+ath79-nand
+~~~~~~~~~~
+
+- GL.iNet
+
+  - GL-XE300
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - EAP615-Wall
+
+- Wavlink
+
+  - WS-WN572HP3 4G
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - TL-MR6400 (v5)
+
+
+Features
+--------
+
+DNS Caching
+~~~~~~~~~~~
+
+DNS-Caching using the dnsmasq resolver is reintroduced.
+
+See the :ref:`DNS-Caching documentation <dns-caching>` section for
+details on how to enable this feature.
+
+Cellular Modem Support
+~~~~~~~~~~~~~~~~~~~~~~
+
+Support for using cellular data connections as the primary uplink connection has been added
+to Gluon. This is supported for hardware that comes with a built-in cellular modem. The required user
+configuration for the APN and SIM-PIN can be performed in the Advanced Settings in Config Mode.
```

Comment:
The Cellular settings are currently only exposed in the config mode UI if the [gluon-web-cellular](https://github.com/freifunk-gluon/gluon/tree/master/package/gluon-web-cellular) package is added manually.

One way to achieve this is to select it as a standard feature:

```make
GLUON_FEATURES_standard := \
	web-cellular \
```

Without this only a configuration via (currently undocumented) uci settings is possible.

Reply:
I'm not quite sure how this should be resolved but i see a few options:

- move my Wiki Entry to docs (i don't like the part about that it's required to add the gluon.cellular section manually) and mention the web-cellular feature
- add the web-cellular feature by default
- suggest adding the web-cellular feature if a community want's to provide that option

I don't like the first two options because they require work and testing. Hoewer, the last and easiest option also isn't elegant and i'm not a fan.

Reply:
Went with the last option.